src/screens/FeedScreen/ContentLink.js

```diff
@@ -88,8 +87,9 @@ const ContentLink = ({
               onCardContentPress,
               score,
               item,
-              heightTopic,
+              heightTopic: normalize(60),
               textHeight,
+              contentHeight: dimen.size.FEED_CONTENT_HEIGHT,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The values of `heightTopic` and `contentHeight` have been updated. The new value of `heightTopic` is now a normalized value of 60, and `contentHeight` is set to `dimen.size.FEED_CONTENT_HEIGHT`. These changes seem to be part of an effort to standardize and optimize layout calculations. However, please ensure that these new values are appropriate for all screen sizes and resolutions.
```diff
-              heightTopic,
-               textHeight,
-               containerStyle: {flex: topics.length > 0 ? 0 : 1}
+               heightTopic: normalize(60),
+               textHeight,
+               contentHeight: dimen.size.FEED_CONTENT_HEIGHT,
+               containerStyle: {flex: topics.length > 0 ? 0 : 1}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->